README.md

```diff
@@ -7,92 +7,34 @@
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/dbt-score.svg)](https://pypi.org/project/dbt-score)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://makeapullrequest.com)
 
-Linter for dbt model metadata.
+## What is `dbt-score`?
 
-## Development
+`dbt-score` is a linter for dbt model metadata.
 
-### Prerequisites
+[dbt](https://getdbt.com/) (Data Build Tool) is a great framework for creating,
+building, organizing, testing and documenting _data models_, i.e. data sets
+living in a database or a data warehouse. Through a declarative approach, it
+allows data practitioners to build data with a methodology inspired by software
+development practices.
 
-You'll need the following prerequisites:
+This leads to data models being bundled with a lot of metadata, such as
+documentation, data tests, access control information, column types and
+constraints, 3rd party integrations... Not to mention any other metadata that
+organizations need, fully supported through the `meta` parameter.
 
-- Any Python version starting from 3.11
-- [pre-commit](https://pre-commit.com/)
-- [PDM](https://pdm-project.org/2.12/)
+At scale, with hundreds or thousands of data models, all this metadata can
+become confusing, disparate, and inconsistent. It's hard to enforce good
+practices and maintain them in continuous integration systems. This is where
+`dbt-score` chimes: by allowing data teams to programatically define and enforce
```

Comment:
`chimes`:

Never heard of this, also unable to find a good definition :thinking:

Reply:
shines?

Reply:
Nice catch, I meant "chimes in" (https://dictionary.cambridge.org/dictionary/english/chime-in) but will replace with "plays its role", it's more clear

Reply:
:+1: